portfolio/src/main/webapp/implinks.html

```diff
@@ -0,0 +1,44 @@
+<!DOCTYPE html>
+<html>
+  <head>
+    <meta charset="UTF-8">
+    <title>Important links</title>
+    <link rel="stylesheet" href="style.css">
+    <script src="script.js"></script>
+    <style>
+        li a {
+        display: block;
+        color: #343;
+        padding: 8px 16px;
+        text-decoration: underline;
+        }
+
+      li a:hover {
+        background-color: yellow;
+        color: purple;
+       }
+
+      ul {
+        padding: 0;
+        margin: 0;
+        width: 200px;
+        background-color:grey;
+         }
+    </style>
+  </head>
+  <body>
+    <div id="content">
+      <h1> Welcome!</h1>
+      <h2 style="font-family:Helvetica;"> Here are some links about myself:</h2>
+      <p><a href="https://www.linkedin.com/in/myles-b-b7623aaa/"<a/>LinkedIn</p>
+      <p><a href="https://my.playstation.com/profile/bosticc"<a/>PlayStation</p>
+      <p><a href="https://instagram.com/bosticc"<a/>Instagram</p>
+      <h3><a href = "index.html"<a/>Go home</h3>
+      <ul>
+        <li><a href="aboutme.html">About me</a></li>
```

Comment:
Different words in file names should be separated by underscores, like: `about_me.html`